lib/controller/page/type/page.summary/page.summary.controller.js

```diff
@@ -164,6 +164,39 @@ SummaryController.attachJsonSubmission = async (submissions, userData) => {
   submissions.push(submission)
 }
 
+SummaryController.attachCsvSubmission = async (submissions, userData) => {
+  const formatEmail = (str, markdown = false) => {
+    try {
+      str = format(str, userData.getScopedUserData(), {markdown, lang: userData.contentLang})
+    } catch (e) {
+      //
+    }
+    return str
+  }
+
+  const serviceName = getInstanceProperty('service', 'name')
+  let subject = getInstanceProperty('service', 'emailSubjectTeam') || `${serviceName} submission`
+  subject = formatEmail(subject)
```

Comment:
Maybe just:
```
const subject = formatEmail(getInstanceProperty('service', 'emailSubjectTeam') || `${serviceName} submission`)
```
Or break it over a few lines to bring attention to the default condition:
```
const subject = formatEmail(
  getInstanceProperty('service', 'emailSubjectTeam') || `${serviceName} submission`
)
```